### DISCUSSION TITLE
Muy buenas

### DISCUSSION THREAD

**Discussion by Amhedriel** [Launch-X-Latam/MisionFrontEnd#476] [Team Posts]
Hola, escribo este mensaje a modo de "confirmación" (Para ver si ya pude ingresar). Saludos y éxitos a todos, abrazos.

**Comment by Josephr1318** [comment]
Si, ya me llego un correo para acceder a mi grupo de teams

**Comment by SebastianDlaCruz** [comment]
si , ya ingrese

**Comment by Amhedriel** [comment]
Yo pude en el 3er intento jeje

**Comment by pachenico-pixel** [comment]
¡Yo también pude!

**Comment by IIDarkTexII** [comment]
.

**Comment by Gilberto-Guzman** [comment]
Igual yo :D

**Comment by carlosayon53** [comment]
Creo que si, ya estoy dentro del grupo

**Comment by AbrahanVIII** [comment]
me reporto, creo que yo también :)

**Comment by JoseEduardoVelazquezCN** [comment]
Estamos dentro🐧:')

**Comment by MercedesMV** [comment]
Reportandome, estamos dentro :)

**Comment by TzipaniLaunchX** [comment]
Igual yo, reportandome, éxito!

**Comment by Sebastian-Longhitano** [comment]
Me reporto!
Éxitos a todos =)

**Comment by dev-crisloli** [comment]
Buenas con todos, cómo se entra al grupo de team envié solicitud pero todavía no le sale

**Comment by ErickOrtiz0298** [comment]
Reportandomee Saludos n.n

**Comment by MisterhayabusaTelcel** [comment]
Reportandome Saludos

**Comment by iSrDiiabloNS** [comment]
Hola a todos, reportándome para confirmar el acceso, saludos y éxitos a todos.

**Comment by recarde** [comment]
Holaaa

**Comment by maxkario** [comment]
Curioso. Pero aquí estoy

**Comment by Mothicc** [comment]
No tiene que reportarse, solo nos llenan de spam.
Porfavor ya no respondan a este correo.

**Comment by CristianAndrey09** [comment]
interesante!!

**Comment by Blan25ga** [comment]
Aparezco en el grupo, pero nunca se agrego el nuevo grupo a teams ni a al mail asociado.
esto esta bien?

**Comment by Amhedriel** [comment]
Buenas **Blan25ga** no entendí muy bien lo que quisiste decir pero, hasta donde sé el nuevo grupo en teams aún no existe el otro grupo de manera formal, porque *BackEnd* lo veremos en 6 semanas, asi que solo está el grupo de *FrontEnd*, porque es lo que estamos estudiando ahora, y con lo del mail, si estas en el primer grupo *FrontEnd* al realizar los pasos que mencionaron los administradores entonces ya estás adentro, si asociaste el primer mail (que es el de *FrontEnd*) entonces no te pedirá que asocies de nuevo con la misma cuenta.

Abrazos

**Comment by Amhedriel** [comment]
Y Mothicc te recomiendo que deshabilites las notificaciones de este hilo y de todos los que no te interesen, porque somos como 2000 estudiantes y hasta ahora ingresamos como 20 en el nuevo grupo, entiendo que te moleste pero te dan la opción de deshabilitarlo.

Saludos

**Comment by Blan25ga** [comment]
Gracias por la respuesta!!!

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
